### PR TITLE
feat(maven): Add working-dir option 

### PR DESCRIPTION
**Pull request by vbshnsk**

### CODE REVIEW

File: maven/action.yml [hunk 1]
```diff
@@ -39,6 +39,9 @@ inputs:
       This value will not be used if `NEXUS_PASSWORD` is already set as an env var.
     default: nexus-password
     required: false
+  working-dir:
```
Comment (sasjo) [inline maven/action.yml:42]: I'd prefer that we use `working-directory` to match the name used by the built-in `run` step.

**Review by sasjo — CHANGES_REQUESTED** [review]
Thanks for the contribution!
I'd prefer to change the name of the input property, otherwise change looks good!

File: maven/action.yml (hunk 1 above)
Comment (vbshnsk) [reply maven/action.yml]: updated ✅

**Review by sasjo — APPROVED** [review]
LGTM

**Pull request merged**